_config.yml

```
@@ -132,46 +133,21 @@ subscribeAction: "https://gdg.us5.list-manage1.com/subscribe/post?u=9fc8aa205b05
 subscribeInfo: "Registration will be open around end of August. Number of tickets are limited. Stay tuned!"
 
 # Tickets Block
-ticketsTitle: "Pricetable"
-ticketsInfo: "Tickets grant access to all conference sections, coffee brakes, lunch and party. Accommodation is NOT included in the ticket price."
+ticketsTitle: "Registration"
+ticketsInfo: 'Tickets grant access to all sessions, lunch on both day and one conference dinner. Accommodation is NOT included in the ticket price. If you would like to apply for a waiver, please send an email to <a href="mailto:secretary@nordic-rse.org">secretary@nordic-rse.org</a>, decision will be made on a per-case basis.'
 ticketsOffers:
+
  -
-  name: "Early Bird"
-  price: "350"
-  priceCurrency: "UAH"
-  ticketsNumber: "150"
-  ribbon: "-5%"
-  buyButtonLink: "http://dfua.ticketforevent.com/"
-  buyButtonText: "Buy Now"
-  ticketContentList: ["Lorem ipsum is simply", "Dummy text of the printing", "Typesetting industry"]
-  validFrom: "2014-08-25T10:00"
-  validThrough: "2014-09-30T23:59"
- -
-  name: "Last Chance"
-  price: "450"
-  priceCurrency: "UAH"
-  ticketsNumber: "100"
-  ribbon: "-20%"
+  name: "Genera admission"
+  price: "900"
+  priceCurrency: "SEK"
   featured: true
-  buyButtonLink: "http://dfua.ticketforevent.com/"
-  buyButtonText: "Buy Now"
-  ticketContentList: ["Lorem ipsum is simply", "Dummy text of the printing", "Typesetting industry"]
-  validFrom: "2014-10-01T00:00"
-  validThrough: "2014-10-23T23:59"
-  disabled: true
- -
-  name: "Student"
-  price: "200"
-  priceCurrency: "UAH"
-  ticketsNumber: "50"
-  buyButtonLink: "http://dfua.ticketforevent.com/"
+  ticketsNumber: ""
+  buyButtonLink: "FIXME"
```

Missing link. What is the syntax?

```suggestion
  buyButtonLink: https://www.tickettailor.com/checkout/new-session/id/5204835/chk/bca4/?ref=website_widget&show_search_filter=true&show_date_filter=true&show_sort=true)
```

---

so actually we are not using this, as I use the ticketaylor widget directly